### PR TITLE
[xc-admin] Add activate button for drafts

### PR DESCRIPTION
**Pull request by guibescos**
A button to activate proposals that are in draft mode. This happens when xc-admin-proposer fails to publish the entire proposal.

Also I add waiting 1 second between retries when broadcast proposal creating transactions.

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_common/src/propose.ts [hunk 1]
```diff
@@ -409,6 +409,7 @@ export class MultisigVault {
             needToFetchBlockhash = true;
           }
           console.log(e);
+          await new Promise((r) => setTimeout(r, 1000));
```
Comment (m30m) [inline governance/xc_admin/packages/xc_admin_common/src/propose.ts:412]: I'd suggest increasing this to 5 seconds, we are not in a hurry and 1 second may not be enough sometimes.

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_common/src/propose.ts:412]: mmm let's try 5 seconds if we still struggle

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_common/src/propose.ts:412]: yes

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_common/src/propose.ts]: Made it 3 seconds for now

**Pull request merged**